tests/read_file.C

```diff
@@ -0,0 +1,13 @@
+{
+  //hipo file to be analysed
+  //Simulated electron + proton -> proton + X + 2pi, 500 events
+  HipoChain chain;
+  chain.Add("./tests/data/EptopX2pi_sample.hipo");
+
+  //check file exists
+  int nfiles = chain.GetNFiles();
+  cout<<nfiles<<endl;
+  if (nfiles==0){
```

Comment:
This is not actually testing anything other than ROOT TChain can return number of files given.

Reply:
Indeed the main work behind this is getting clas12root to build with all dependencies, this is here as a starting point. As discussed it would make sense to recreate the RunRoot examples, however the output needs to get something that we can check i.e number of events, fitted peak in a place we expect, an example of using the database to get values (which are known and can be verified). Those when then be the conditions of the test.